config.xml

```diff
@@ -3,7 +3,7 @@
     <name>Stockpile</name>
     <description>An app that manages stuff for organizations</description>
     <author email="info@stockpileapp.co" href="https://stockpileapp.co">Stockpile Team</author>
-    <content src="index.html" />
+    <content original-src="index.html" src="http://172.26.74.227:8101" />
```

Comment:
I think it was generated because I ran the emulator with live reload. Building the app again without live reload removed it,